src/app/query/StoreConfig.query.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+import { Field } from 'Util/Query';
+
+class StoreConfigQuery {
+    getQuery(options) {
+        if (!options) throw new Error('Missing argument `options`');
+        const { configs } = options;
+        return new Field('storeConfig')
+            .addFieldList(configs);
+    }
+}
+
```

Comment:
Could you please add `export { StoreConfigQuery };` we do from version `1.8.1` in order to make Query extension easier.